content/news/index.md

```diff
@@ -0,0 +1,21 @@
+---
+news:
```

Comment:
All the news items to be displayed in the `/news` page can be added here following the same format